go.mod

```diff
@@ -127,8 +127,8 @@ require (
 	github.com/shirou/gopsutil v2.20.3+incompatible
 	github.com/stretchr/testify v1.5.1
 	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
-	github.com/tron-us/go-btfs-common v0.3.7
-	github.com/tron-us/go-common/v2 v2.0.5
+	github.com/tron-us/go-btfs-common v0.5.0-pre4
```

Comment:
Please review on https://github.com/TRON-US/go-btfs-common/pull/118 for proto chanages of Escrow and Guard.